src/app/components/core/user-management/user.management.module.ts

```diff
@@ -37,7 +39,7 @@ const routes: Routes = [
     NgProgressModule,
     AlertDialogModule
   ],
-  providers: [],
+  providers: [UserGuard],
```

Comment:
AuthGuard is not needed here?

Reply:
No because AuthGuard is used with other modules as well so it is inside app.module.ts